youtube_dl/extractor/prosiebensat1.py

```diff
@@ -375,7 +375,9 @@ def _extract_clip(self, url, webpage):
         title = self._html_search_regex(self._TITLE_REGEXES, webpage, 'title')
         info = self._extract_video_info(url, clip_id)
         description = self._html_search_regex(
-            self._DESCRIPTION_REGEXES, webpage, 'description', fatal=False)
+            self._DESCRIPTION_REGEXES, webpage, 'description', default=None, fatal=False)
+        if description is None: 
+            description = self._og_search_description(webpage, default=None)
```

Comment:
default=None vs. fatal=False: the former is silent while the latter issues a warning is matching fails. The first try should be silent and the second try should issue a warning. Anyway don't use default=None and fatal=False in a single call.